assets/scripts/admin/statistics.js

```diff
@@ -590,6 +595,28 @@ LS.Statistics2 = function () {
     $(".stats-showpie").click(function () {
         changeGraphType('showpie', this.parentNode);
     });
+
+    var ajaxDownloadStats = function (url, data) {
```

Comment:
Maybe we can turn this into a generic function that returns the xhr object, so anyone can continue later adding behaviour on error or success